bin/addons-linter.sh

```diff
@@ -9,27 +9,9 @@
 
 . $(dirname $0)/commons.sh
 
-TMPDIR=/tmp/MAC_addonsLinter
-
 print Y "Update the submodules..."
 git submodule init || die
 git submodule update --remote --depth 1 src/_locales || die
 
-printn Y "Removing previous execution data... "
-rm -rf $TMPDIR || die
-print G "done."
-
-printn Y "Creating a tmp folder ($TMPDIR)... "
-mkdir $TMPDIR || die
-print G "done."
-
-printn Y "Copying data... "
-cp -r src $TMPDIR || die
-print G "done."
-
-printn Y "Removing the github folder... "
-rm -rf $TMPDIR/src/_locales/.github || die
-print G "done."
-
```

Comment:
You originally added this to avoid the following linting failure:

```
Code                          Message                    Description                                        File            Line   Column
NO_MESSAGES_FILE_IN_LOCALES   Empty language directory   messages.json file missing in "_locales/.github"   manifest.json
```

Do you think it's fine to let the linter complain about it or would there be a cleaner solution to avoid this error?